### DISCUSSION TITLE
suppress mdn message in `HTTPStatusError`?

### DISCUSSION THREAD

**Discussion by trim21** [encode/httpx#3193] [Ideas]
it doesn't look very nice in error message with 2 urls and seprated lines, consider a flag (on client, maybe?) to disable this part of message?

**Comment by lovelydinosaur** [comment]
Your observation is valid, although against an older version. Is now clean as suggested.

**Reply by trim21** [reply]
I didn't understand, what do you mean `clean as suggested`? I mean this line and it's still here.

https://github.com/encode/httpx/blob/fa6dac8383541669f42086d4f07c71b0f8d6a279/httpx/_models.py#L751
